packages/types/src/explorers/explorer.ts

```diff
@@ -1,4 +1,7 @@
-export enum ExplorersExplorer {
-  CardanoScan = 'cardanoscan',
-  CExplorer = 'cexplorer',
-}
+export const ExplorersExplorer = {
```

Comment:
for non mutating objs I'd rather go with enums, since it is transpiled as fn, anyway.

Reply:
I don't think enums can be listed programmatically, like `Object.values(Explorers)`